main-unix.c

```diff
@@ -461,12 +462,11 @@ main (int argc,
 
         /* Print return value */
         const jerry_value_t args[] = { ret_val };
-        jerry_value_t ret_val_print;
-        if (jerry_call_function (jerry_get_object_value (print_function),
-                                 NULL,
-                                 &ret_val_print,
-                                 args,
-                                 1))
+        jerry_value_t ret_val_print = jerry_call_function (jerry_get_object_value (print_function),
+                                                           NULL,
+                                                           args,
+                                                           1);
+        if (!jerry_value_is_error (ret_val_print))
```

Comment:
`ret_val_print` should also be released when it's an error, otherwise it may leak.